cmd/extensions/installed/cmd.go

```diff
@@ -0,0 +1,34 @@
+package installed
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+)
+
+// InstalledCmd represents the installed command
+var InstalledCmd = &cobra.Command{
+	Use:   "installed",
```

Comment:
That is why I was recommending to check kube lib.
Installed might be simple list and remote would be search comamnd